src/WebUI/src/components/app/InstallationGuide.vue

```diff
@@ -37,82 +37,83 @@ watch(
     </slot>
 
     <template #popper>
-      <div class="space-y-10 py-10">
-        <div class="prose prose-invert space-y-10 px-12">
-          <h3 class="text-center">{{ $t('installation.title') }}</h3>
+      <div class="prose prose-invert space-y-6 px-12 py-10">
+        <h3 class="text-center">{{ $t('installation.title') }}</h3>
 
-          <OTabs v-model="tabsModel" size="xl" :animated="false">
-            <OTabItem
-              :label="$t(`platform.${Platform.Steam}`)"
-              :icon="platformToIcon[Platform.Steam]"
-              :value="PossibleValues.Steam"
-            >
-              <div class="space-y-6">
-                <ol>
-                  <i18n-t scope="global" keypath="installation.platform.steam.subscribe" tag="li">
-                    <template #steamWorkshopsLink>
-                      <a
-                        target="_blank"
-                        href="steam://openurl/https://steamcommunity.com/sharedfiles/filedetails/?id=2878356589"
-                      >
-                        Steam Workshops
-                      </a>
-                    </template>
-                  </i18n-t>
-                  <li>{{ $t('installation.common.bannerlordLauncher') }}</li>
-                  <li>{{ $t('installation.common.multiplayerModsTab') }}</li>
-                  <li>{{ $t('installation.common.activateMod') }}</li>
-                  <li>{{ $t('installation.common.launchMultiplayerGame') }}</li>
-                </ol>
-                <p class="text-content-400">{{ $t('installation.platform.steam.update') }}</p>
-              </div>
-            </OTabItem>
+        <OTabs v-model="tabsModel" size="xl" :animation="['fade', 'fade']">
+          <OTabItem :value="PossibleValues.Other">
+            <template #header>
+              <OIcon :icon="platformToIcon[Platform.Steam]" size="xl" />
+              <OIcon :icon="platformToIcon[Platform.Microsoft]" size="xl" />
+              <OIcon :icon="platformToIcon[Platform.EpicGames]" size="xl" />
+              {{ $t('installation.platform.other.title') }}
+            </template>
+            <ol>
+              <i18n-t
+                scope="global"
+                keypath="installation.platform.other.downloadLauncher"
+                tag="li"
+              >
+                <template #launcherLink>
+                  <a target="_blank" href="https://c-rpg.eu/LauncherV3.exe">Launcher</a>
+                </template>
+              </i18n-t>
+              <li>{{ $t('installation.platform.other.startLauncher') }}</li>
+              <li>{{ $t('installation.common.bannerlordLauncher') }}</li>
+              <li>{{ $t('installation.common.multiplayerModsTab') }}</li>
+              <li>{{ $t('installation.common.activateMod') }}</li>
+              <li>{{ $t('installation.common.launchMultiplayerGame') }}</li>
```

Comment:
This isn't the process for using the launcher, after the game has updated, when they 'Launch CRPG' it should start bannerlord for them already in multiplayer, without having to worry about what mods to load & their load order,


![image](https://github.com/namidaka/crpg/assets/93069911/6ecf5228-87f3-45f4-a0f2-8f6b6e868e3a)

Reply:
Quite right actually, I forgot that our Launcher process had changed around and didn't even think to touch that. I'll get this updated and will submit

Reply:
Updated now